platform/ios/resources/lt.lproj/Localizable.stringsdict

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>MAP_A11Y_VALUE</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@level@
+%#@count@</string>
+		<key>level</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>d</string>
+			<key>one</key>
+			<string>Priartinimas: %dx</string>
+			<key>few</key>
+			<string>Priartinimas: %dx</string>
+			<key>other</key>
+			<string>Priartinimas: %dx</string>
+		</dict>
+		<key>count</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>ld</string>
+			<key>one</key>
+			<string>Matomos anotacijos: %d anotacija</string>
+			<key>few</key>
+			<string>Matomos anotacijos: %d anotacijos</string>
+			<key>other</key>
+			<string>Matomos anotacijos: %d anotacijų</string>
```

Comment:
@eimantas, FYI, “annotation” wound up being duplicated here. I don’t know how to decline in these cases, so please feel free to fix them in Transifex and I’ll pull the updates into this PR.